packages/web-pkg/src/components/SideBar/WebDavDetails.vue

```diff
@@ -38,24 +38,27 @@
 </template>
 
 <script lang="ts">
-import { defineComponent, inject, ref, Ref, computed, unref } from 'vue'
+import { defineComponent, inject, ref, Ref, computed, unref, PropType } from 'vue'
 import { urlJoin } from '@ownclouders/web-client/src/utils'
-import { Resource } from '@ownclouders/web-client'
-import { useConfigStore } from '../../composables'
+import { Resource, SpaceResource } from '@ownclouders/web-client'
 
 export default defineComponent({
   name: 'WebDavDetails',
-  setup() {
-    const configStore = useConfigStore()
-
+  props: {
+    space: {
+      type: Object as PropType<SpaceResource>,
+      required: true
+    }
+  },
+  setup(props) {
     const resource = inject<Ref<Resource>>('resource')
     const copiedIcon = 'check'
     const copyIcon = 'file-copy'
     const copyWebDAVPathIcon = ref(copyIcon)
     const copyWebDAVUrlIcon = ref(copyIcon)
 
     const webDavUrl = computed(() => {
-      return urlJoin(configStore.serverUrl, unref(resource).webDavPath)
+      return urlJoin(props.space.root.webDavUrl, unref(resource).path)
```

Comment:
Thanks 👍 We should forward port current `stable-8.0` right after this PR is merged, so that we still have in mind how to solve the merge conflict. :-)